SonarQube.TeamBuild.PreProcessor/Program.cs

```diff
@@ -23,8 +23,8 @@ private static int Main(string[] args)
             Utilities.LogAssemblyVersion(logger, typeof(Program).Assembly, Resources.AssemblyDescription);
             logger.IncludeTimestamp = true;
 
-            TeamBuildPreProcessor preProcessor = new TeamBuildPreProcessor();
-            bool success = preProcessor.Execute(args, logger);
+            TeamBuildPreProcessor preProcessor = new TeamBuildPreProcessor(logger);
+            bool success = preProcessor.Execute(args);
```

Comment:
I modified the pre-processor to accept the logger in the constructor so it can pass the logger on to the analyser provider when it is constructed: see below.